diff --git public/app/features/panel/panel_directive.ts public/app/features/panel/panel_directive.ts
index 4637f0169..24977bd38 100644
--- public/app/features/panel/panel_directive.ts
+++ public/app/features/panel/panel_directive.ts
@@ -156,7 +156,7 @@ module.directive('grafanaPanel', function($rootScope) {
             content: function() {
               return ctrl.getInfoContent({mode: 'tooltip'});
             },
-            position: 'top center',
+            position: 'right middle',
             classes: ctrl.error ? 'drop-error' : 'drop-help',
             openOn: 'hover',
             hoverOpenDelay: 100,
diff --git public/app/plugins/panel/singlestat/module.ts public/app/plugins/panel/singlestat/module.ts
index 23283e985..0c5d72c72 100644
--- public/app/plugins/panel/singlestat/module.ts
+++ public/app/plugins/panel/singlestat/module.ts
@@ -566,7 +566,7 @@ class SingleStatCtrl extends MetricsPanelCtrl {
         if (!linkInfo) { return;}
 
         drilldownTooltip.text('click to go to: ' + linkInfo.title);
-        drilldownTooltip.place_tt(e.pageX, e.pageY-50);
+        drilldownTooltip.place_tt(e.pageX+20, e.pageY-15);
       });
     }
 
