diff --git a/public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx b/public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx
index 13f22fc7d4..df671890c8 100644
--- a/public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx
+++ b/public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx
@@ -1,5 +1,5 @@
 import React, { PureComponent } from 'react';
-import { Button, ClipboardButton, Input, LinkButton, Select } from '@grafana/ui';
+import { Button, ClipboardButton, Input, Select } from '@grafana/ui';
 import { AppEvents, SelectableValue } from '@grafana/data';
 import { getBackendSrv } from '@grafana/runtime';
 import { DashboardModel, PanelModel } from 'app/features/dashboard/state';
@@ -13,6 +13,7 @@ const expireOptions: Array<SelectableValue<number>> = [
   { label: '1 Hour', value: 60 * 60 },
   { label: '1 Day', value: 60 * 60 * 24 },
   { label: '7 Days', value: 60 * 60 * 24 * 7 },
+  { label: '90 Days', value: 60 * 60 * 24 * 90 },
 ];
 
 interface Props {
@@ -43,10 +44,10 @@ export class ShareSnapshot extends PureComponent<Props, State> {
     this.state = {
       isLoading: false,
       step: 1,
-      selectedExpireOption: expireOptions[0],
-      snapshotExpires: expireOptions[0].value,
+      selectedExpireOption: expireOptions[4],
+      snapshotExpires: expireOptions[4].value,
       snapshotName: props.dashboard.title,
-      timeoutSeconds: 4,
+      timeoutSeconds: 30,
       snapshotUrl: '',
       deleteUrl: '',
       externalEnabled: false,
@@ -209,13 +210,34 @@ export class ShareSnapshot extends PureComponent<Props, State> {
       <>
         <div>
           <p className="share-modal-info-text">
-            A snapshot is an instant way to share an interactive dashboard publicly. When created, we{' '}
-            <strong>strip sensitive data</strong> like queries (metric, template and annotation) and panel links,
-            leaving only the visible metric data and series names embedded into your dashboard.
+            A snapshot is a way to securely share your dashboard with Percona. When created, we{' '}
+            <strong>strip sensitive data</strong>
+            like queries (metrics, template variables, and annotations) along with panel links. The shared dashboard
+            will only be available for viewing by Percona Engineers, and the content on the dashboard will assist
+            Percona Engineers in troubleshooting your case.
           </p>
           <p className="share-modal-info-text">
-            Keep in mind, your <strong>snapshot can be viewed by anyone</strong> that has the link and can reach the
-            URL. Share wisely.
+            You can safely leave the defaults set as they are, but for further information:
+          </p>
+          <p className="share-modal-info-text">
+            <ul>
+              <li>
+                <strong>Snapshot Name:</strong> The name Percona will see when viewing your dashboard.
+              </li>
+              <li>
+                <strong>Expire:</strong> How long before snapshot should expire, configure lower if required. Percona
+                automatically purges shared dashboards after 90 days.
+              </li>
+              <li>
+                <strong>Timeout (seconds):</strong> Duration the dashboard will take to load before the snapshot is
+                generated.
+              </li>
+            </ul>
+          </p>
+          <p className="share-modal-info-text">
+            <strong>What to do next:</strong>
+            After clicking Share with Percona, wait for the dashboard to be generated, and you will be provided a unique
+            URL that then needs to be communicated to Percona via your ticket.
           </p>
         </div>
         <div className="gf-form-group share-modal-options">
@@ -273,13 +273,6 @@ export class ShareSnapshot extends PureComponent<Props, State> {
             </ClipboardButton>
           </div>
         </div>
-
-        <div className="pull-right" ng-if="step === 2" style={{ padding: '5px' }}>
-          Did you make a mistake?{' '}
-          <LinkButton variant="link" target="_blank" onClick={this.deleteSnapshot}>
-            delete snapshot.
-          </LinkButton>
-        </div>
       </>
     );
   }
diff --git a/public/sass/components/_gf-form.scss b/public/sass/components/_gf-form.scss
index eb6e0be596..c1de97a13e 100644
--- a/public/sass/components/_gf-form.scss
+++ b/public/sass/components/_gf-form.scss
@@ -101,6 +101,10 @@ $input-border: 1px solid $input-border-color;
   }
 }
 
+.gf-form-button-row-custom {
+  padding-top: 1rem;
+}
+
 .gf-form-label {
   padding: $input-padding;
   flex-shrink: 0;
diff --git a/public/sass/components/_modals.scss b/public/sass/components/_modals.scss
index 3af632a4ce..2f1960e54e 100644
--- a/public/sass/components/_modals.scss
+++ b/public/sass/components/_modals.scss
@@ -137,7 +137,7 @@
   }
 
   .share-modal-options {
-    margin: 11px 0px 33px 0px;
+    margin: 11px 0px 25px 0px;
     display: inline-block;
   }
 
